test.js

```diff
@@ -209,9 +210,11 @@ describe('easy stackup function', function() {
   // NOTE: (mc) Perhaps instead of these tests, we should check that we're passing
   // the correct things to gerber-to-svg, whats-that-gerber, and pcb-stackup-core?
   it('has deterministic top and bottom svgs if ids are given', function(done) {
+    var exampleGerber1 = fs.createReadStream('./integration/boards/arduino-uno/arduino-uno.gko')
+    var exampleGerber2 = fs.createReadStream('./integration/boards/arduino-uno/arduino-uno.plc')
```

Comment:
I think './'  and forward slashes in general will work across platforms. Aren't these tests supposed to run in browsers as well though?